### ISSUE TITLE
Problem with installation pyvrp under Windows 11

### ISSUE THREAD

**Issue by HJP-HPS** [PyVRP/PyVRP#674]
I have a problem installing pyvrp, Perhaps i can get help - Thank you.

The installaion via pip install pyvrp fails with the following notice

```
Building wheels for collected packages: pyvrp
  Building wheel for pyvrp (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for pyvrp (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [77 lines of output]
      The Meson build system
      Version: 1.6.0
      Source dir: C:\Users\XXXXXXXX\AppData\Local\Temp\pip-install-xgjuqs5f\pyvrp_bd4cbe5808d944868d914f48c6a72fd6
      Build dir: C:\Users\XXXXXXXX\AppData\Local\Temp\pip-install-xgjuqs5f\pyvrp_bd4cbe5808d944868d914f48c6a72fd6\build
      Build type: native build
      Project name: pyvrp
      Project version: 0.9.1

      WARNING: Failed to activate VS environment: Could not find C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe

      meson.build:1:0: ERROR: Unknown compiler(s): [['icl'], ['cl'], ['c++'], ['g++'], ['clang++'], ['clang-cl']]
      The following exception(s) were encountered:
      Running `icl ""` gave "[WinError 2] Das System kann die angegebene Datei nicht finden"
      Running `cl /?` gave "[WinError 2] Das System kann die angegebene Datei nicht finden"
      Running `c++ --version` gave "[WinError 2] Das System kann die angegebene Datei nicht finden"
      Running `g++ --version` gave "[WinError 2] Das System kann die angegebene Datei nicht finden"
      Running `clang++ --version` gave "[WinError 2] Das System kann die angegebene Datei nicht finden"
      Running `clang-cl /?` gave "[WinError 2] Das System kann die angegebene Datei nicht finden"

      A full log can be found at C:\Users\XXXXXXXX\AppData\Local\Temp\pip-install-xgjuqs5f\pyvrp_bd4cbe5808d944868d914f48c6a72fd6\build\meson-logs\meson-log.txt
```

The full output 
[python_ pip install  pyvrp.txt](https://github.com/user-attachments/files/17632923/python_.pip.install.pyvrp.txt)
is in the encloure

**Comment by N-Wouda** [comment]
Hi @HJP-HPS, you appear to be building from source for Windows. We offer pre-compiled binaries as well, but apparently none that match your platform. Could you attach the output of the following command to your post?
```
python -c 'import pyvrp; pyvrp.show_versions()'
```

**Comment by N-Wouda** [comment]
Ah, I see in your attached output that it's Python 3.13. We did not yet ship pre-compiled binaries for 3.13 with PyVRP 0.9.1. We do build them now, but haven't released v0.10.0 yet to the package index. We will do so soon. For now, you can install pre-compiled binaries from our CD action, [here](https://github.com/PyVRP/PyVRP/actions/runs/11594334971). Simply download the Windows distribution, and install the wheel for Python 3.13 (`pip install <path to wheel>`).

Once we release PyVRP 0.10.0, a pre-built wheel will be available on the package index, and a regular `pip install pyvrp` should work again with Python 3.13.

**Comment by HJP-HPS** [comment]
Hello Nils,
Thank you for the quick reply. I will try it out tomorrow and I will let you know. 


Regards, 

Hans, HPS GmbH 


Von unterwegs gesendet.

Am 05.11.2024 um 14:23 schrieb Niels Wouda ***@***.***>:




﻿

Ah, I see in your attached output that it's Python 3.13. We did not yet ship pre-compiled binaries for 3.13 with PyVRP 0.9.1 (it wasn't out yet when we released 0.9.1). We do build them now, but haven't released v0.10.0 yet to the package index.
 We will do so soon. For now, you can install pre-compiled binaries from our CD action,
here. Simply download the Windows distribution, and install the wheel for Python 3.13 (pip install <path to wheel>).
Once we release PyVRP 0.10.0, a pre-built wheel will be available on the package index, and a regular
pip install pyvrp should work again with Python 3.13.
—
Reply to this email directly, 
view it on GitHub, or 
unsubscribe.
You are receiving this because you were mentioned.Message
 ID: ***@***.***>

**Comment by HJP-HPS** [comment]
Hi Nils,

thank you very much for your tip, it worked and we were able to run the first examples successfully.

Regards,

Hans

Am 05.11.2024 um 14:23 schrieb Niels Wouda ***@***.***>:
﻿

Ah, I see in your attached output that it's Python 3.13. We did not yet ship pre-compiled binaries for 3.13 with PyVRP 0.9.1 (it wasn't out yet when we released 0.9.1). We do build them now, but haven't released v0.10.0 yet to the package index. We will do so soon. For now, you can install pre-compiled binaries from our CD action, here<https://github.com/PyVRP/PyVRP/actions/runs/11594334971>. Simply download the Windows distribution, and install the wheel for Python 3.13 (pip install <path to wheel>).

Once we release PyVRP 0.10.0, a pre-built wheel will be available on the package index, and a regular pip install pyvrp should work again with Python 3.13.

**Comment by N-Wouda** [comment]
Fantastic! Glad to hear you got everything working. We have now also released PyVRP 0.10.0 to the Python package index, so `pip install pyvrp` should also work again.
